parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -103,6 +103,11 @@ public static Metrics footerMetrics(ParquetMetadata metadata, MetricsConfig metr
     for (BlockMetaData block : blocks) {
       rowCount += block.getRowCount();
       for (ColumnChunkMetaData column : block.getColumns()) {
+        if (column.getPrimitiveType().getPrimitiveTypeName() == PrimitiveType.PrimitiveTypeName.INT96) {
+          // skip metrics for INT96 timestamps
```

Comment:
We may want to include a justification for skipping: the stats are not reliable.

Reply:
Will do

Reply:
Done